Keyed/Incidents.xml

```diff
@@ -56,40 +56,40 @@
   <AwokeDaysAgo>Проснулся {0} дней назад</AwokeDaysAgo>
-  <PsychicDroneLevel>Психозонд, уровень: {0}</PsychicDroneLevel>
+  <PsychicDroneLevel>Гул, уровень {0}</PsychicDroneLevel>
   <PsychicDroneLevelLow>низкий</PsychicDroneLevelLow>
   <PsychicDroneLevelMedium>средний</PsychicDroneLevelMedium>
   <PsychicDroneLevelHigh>высокий</PsychicDroneLevelHigh>
   <PsychicDroneLevelExtreme>сверхвысокий</PsychicDroneLevelExtreme>
-  <LetterLabelPsychicDroneLevelIncreased>раздражение усилилось</LetterLabelPsychicDroneLevelIncreased>
-  <LetterPsychicDroneLevelIncreased>От части разбившегося корабля теперь исходит ещё более мощный вид психической активности. Всякий человек в этой местности теперь чувствует себя ещё более обеспокоенным, напуганным и параноидальным.</LetterPsychicDroneLevelIncreased>
+  <LetterLabelPsychicDroneLevelIncreased>гул усилился</LetterLabelPsychicDroneLevelIncreased>
+  <LetterPsychicDroneLevelIncreased>От части разбившегося корабля теперь исходит более мощный вид психического гула. Всякий человек в этой местности теперь ещё более обеспокоен, напуган и мнителен.</LetterPsychicDroneLevelIncreased>
 
   <!-- Agent revealed -->
 	<LetterLabelAgentRevealed>Шпион</LetterLabelAgentRevealed>
-	<LetterAgentRevealed>Вражеский шпион {0} из {1} раскрыт.\n\nСейчас HE пытается бежать из поселения.</LetterAgentRevealed>
+	<LetterAgentRevealed>Вражеский шпион {0} из фракции {1} раскрыт.\n\nСейчас HE пытается бежать из поселения.</LetterAgentRevealed>
 
   <!-- Distant psychic drone -->
-  <LetterLabelPsychicDrone>Раздражение</LetterLabelPsychicDrone>
-  <LetterIncidentPsychicDrone>Головная боль пронзает черепа ваших поселенецев словно гвоздь!\n\nИзлучение некого древнего бесчеловечного психического зонда пронизывает эту местность. В течение нескольких дней настроение у некоторых людей ({0}) будет немного хуже.</LetterIncidentPsychicDrone>
+  <LetterLabelPsychicDrone>Гул</LetterLabelPsychicDrone>
+  <LetterIncidentPsychicDrone>Головная боль пронзает черепа ваших поселенцев словно гвоздь!\n\nИзлучение некоего древнего бесчеловечного психического зонда пронизывает эту местность. В течение нескольких дней настроение у некоторых людей ({0}) будет немного хуже.</LetterIncidentPsychicDrone>
 
-  <LetterLabelPsychicSoothe>Успокоение</LetterLabelPsychicSoothe>
-  <LetterIncidentPsychicSoothe>Волна блаженства проносится по всему поселению!\n\nИзлучение некого древнего генератора счастья пронизывает эту область. В течение нескольких дней настроение у некоторых людей ({0}) будет немного лучше.</LetterIncidentPsychicSoothe>
+  <LetterLabelPsychicSoothe>Умиротворение</LetterLabelPsychicSoothe>
+  <LetterIncidentPsychicSoothe>Волна блаженства проносится по всему поселению!\n\nИзлучение некоего древнего генератора счастья пронизывает эту область. В течение нескольких дней настроение у некоторых людей ({0}) будет немного лучше.</LetterIncidentPsychicSoothe>
```

Comment:
Some distant engine of happiness is stirring. It is projecting a powerful psychic drone over this entire region, on a frequency that only seems to affect the {0} **gender**. For a few days, some people's mood will be quite a bit better.